include/logging/log_core.h

```diff
@@ -895,6 +896,31 @@ do {									       \
 
 #include <syscalls/log_core.h>
 
+/** @brief Function pointer table for log output formats, indexed by enum log_format_flags.
```

Comment:
Outdated comment; log_format_flags does not exist

Reply:
Addressed.